todo_app/tests/test_view_model.py

```diff
@@ -0,0 +1,56 @@
+"""Test module to test ViewModel class"""
+import pytest
+
+from todo_app.data.view_model import ViewModel
+
+
+@pytest.fixture
+def items():
+    return [{'name': 'item1', 'id': "abc1", 'status': 'To do'},
+            {'name': 'item2', 'id': "abc2", 'status': 'To do'},
+            {'name': 'item3', 'id': "abc3", 'status': 'Doing'},
+            {'name': 'item4', 'id': "abc4", 'status': 'Done'},
+            ]
+
+
+def test_view_model_done_property(items):
+    # arrange
+    model_view = ViewModel(items)
+
+    # act
+    done_items = model_view.done_items
+
+    # assert
+    expected = [{'name': 'item4', 'id': "abc4", 'status': 'Done'},]
+
+    assert len(done_items) == 1
+    assert done_items == expected
+
+
+def test_view_model_doing_property(items):
+    # arrange
+    model_view = ViewModel(items)
```

Comment:
Given that this step is duplicated in each test, we could always make the `items` fixture into a `view_model` fixture and just have that passed through, but it's a very minor change